apm-agent-plugins/apm-grpc/apm-grpc-test-latest/pom.xml

```diff
@@ -21,6 +21,30 @@
         <protobuf.version>3.11.0</protobuf.version>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>versions-maven-plugin</artifactId>
+                <version>2.7</version>
+                <executions>
+                    <execution>
+                        <phase>initialize</phase>
+                        <goals>
+                            <!-- because grpc version is set using a shared property -->
+                            <goal>update-properties</goal>
```

Comment:
When setting `grpc.version` to `[1.29.0,)`, do we still need the `versions-maven-plugin`?

Reply:
Yes, it's even simpler without the plugin.

The only difference is that we don't have to update the pom.xml as it automatically (and silently) picks the latest version.